### ISSUE TITLE
Accessibility issue after you click on the link in the search button

### ISSUE THREAD

**Issue by mgifford** [squidfunk/mkdocs-material#6434]
### Context

Related to https://github.com/squidfunk/mkdocs-material/issues/6433

After you click or tab to the search div (this is bad, should be a focusable element) on this page:
https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/

The button should have some meaning. Right now it is empty. 

### Bug description

Title: WCAG 4.1.2: Ensures buttons have discernible text (.t)
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: Reporting a bug - Material for MkDocs - https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/

Element path: .t

Snippet: <button class="t md-search__icon md-icon">

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 119.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- [Reporting a bug - where to see the bug in action](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- 

### Reproduction

![Screenshot of the error](https://github.com/squidfunk/mkdocs-material/assets/116832/fce39fde-bbec-4971-a7c7-27f16f100ae7)

Go to https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug and tab to the search then run Accessibility Insights. 

Full page errors: 
FastPass_20231204_ReportingabugMateria (1).html.zip

### Steps to reproduce

Start with the demo page and run accessibility checkers like Accessibility Insights. 

### Browser

Edge

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting! The chat bot is only a temporary test which we'll remove in the coming weeks. It's not part of Material for MkDocs, it's only part of our deployment. Thus, please ignore this button ☺️
